plutarch-extra/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
+## 1.4.0 -- 07-06-2024
```

Comment:
Last PR made it 1.7.0. So it should be 1.8.0?

Reply:
Yes, it should be. Not sure what happened there.